.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
+branch = True
+parallel = True
+source =
+    lms
+    tests/unit
+
+[paths]
+source =
+    src/
```

Comment:
Is `src/` wrong here? No `src` dir in lms

Reply:
This block is saying that `src/` is equivalent to `*/site-packages/` which allows coverage to match when it's installed as a module.

So this is both wrong and redundant in the case of an app.